product/opni/components/Cluster.vue

```diff
@@ -152,7 +318,22 @@ export default {
   width: 100%;
 }
 
-::v-deep .warning {
+.placeholder-text {
+  font-family: sans-serif !important;
```

Comment:
!important should be used sparingly. If you can make the scss selector more specific you can most likely accomplish the same thing.